tutorials/docker-gcplogs-driver/index.md

```diff
@@ -53,35 +54,30 @@ automatic system updates.
 1.  After the instance is created, click the **SSH** button to open a terminal
     connected to the machine.
 1.  To use the [Google Cloud logging driver for
-    Docker](https://docs.docker.com/engine/admin/logging/gcplogs/), specify the
+    Docker](https://docs.docker.com/config/containers/logging/gcplogs/), specify the
     `--log-driver=gcplogs` command-line argument to the `docker run` command.
 
     Run the following command to start an NGINX container which writes logs to
     Stackdriver Logging.
 
-        docker run -d --name mysite --log-driver=gcplogs -p 80:80 nginx
-
-    You can specify additional options with the `--log-opt` command-line
-    argument. This command tells Docker to also log the command that the container
-    was started with.
-
         docker run -d \
             --name mysite \
             --log-driver=gcplogs \
-            --log-opt gcp-log-cmd=true \
```

Comment:
In my testing today, I couldn't get this to actually have any effect, so I've removed it.